src/index.ts

```diff
@@ -195,18 +231,30 @@ export function HTPipe<
     T4,
     PipedPreContext<T3, PipedPreContext<T2, T1>>
   > &
+    ClashlessSanitizeParams<
+      T4,
+      PipedSanitizeParams<T3, PipedSanitizeParams<T2, T1>>
+    > &
     ClashlessAttachData<T4, PipedAttachData<T3, PipedAttachData<T2, T1>>>,
   T3 extends ClashlessInitPreContext<T3, PipedPreContext<T2, T1>> &
```

Comment:
- [ ] typo below (AttachData instead of SanitizeParams)

Reply:
done